index.html

```diff
@@ -17,12 +17,25 @@
   <header class="page-header">
     <nav>
       <a class="logo-link" href="https://github.com/Khawanin">Khawanin</a>
-      <a href="#"><img class="union-icon" src="./Images/Union.png" alt="union icon"></a>
+      <a href="#"><img id="hamburger-icon" class="union-icon" src="./Images/Union.png" alt="union icon"></a>
       <ul class="desktop-nav">
         <li><a class="nav-list" href="">Portolio</a></li>
         <li><a class="nav-list" href="">About</a></li>
         <li><a class="nav-list" href="">Contact</a></li>
       </ul>
+
+      <!-- close icon-->
+      <a href="#" class="close-icon hidden">
+        <img src="./Images/close.png" alt="close icon">
+      </a>
+
+      <!-- mobile menu -->
+      <ul class="mobile-menu hidden">
+        <li class="mobile-list"><a href="#portfolio">Portfolio</a></li>
+        <li class="mobile-list"><a href="#about-myself">About</a></li>
+        <li class="mobile-list"><a href="#contact-me">Contact</a></li>
+      </ul>
+
     </nav>
```

Comment:
- [ ] Good job so far your mobile menu is good :+1:  but there are a few things you need to work on about your menu design. for example, the Xmark icon to close the menu is missing. Also, there is too much spacing between your menu option which makes it different from the Figma template. Please make sure to fix your menu design and make it similar to the Figma template. As a developer it is very important to pay attention to details :+1: 

#### Your design
![Screenshot from 2022-11-14 22-32-27](https://user-images.githubusercontent.com/6615394/201784067-ea54941e-a757-496a-af89-77c2d6459f49.jpg)

#### Figma template
![Screenshot from 2022-11-14 22-33-40](https://user-images.githubusercontent.com/6615394/201784072-3a1a37ac-9ade-476f-a9b2-3d5fafc37506.jpg)